webknossos-tracingstore/app/com/scalableminds/webknossos/tracingstore/TSRemoteWebknossosClient.scala

```diff
@@ -104,7 +104,7 @@ class TSRemoteWebknossosClient @Inject()(
 
   def createTracingFor(annotationId: String,
                        layerParameters: AnnotationLayerParameters,
-                       previousVersion: Long): Fox[Either[SkeletonTracing, VolumeTracing]] = {
+                       previousVersion: Long): Fox[Either[(SkeletonTracing, Set[Int]), VolumeTracing]] = {
```

Comment:
Yeah maybe a definition for `(SkeletonTracing, Set[Int])` or even `Either[(SkeletonTracing, Set[Int]), VolumeTracing]` might be helpful as it appears quite a lot :see_no_evil: